gps/internal/bitbucketapi/bitbucketapi.go

```diff
@@ -0,0 +1,142 @@
+// Copyright 2019 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package bitbucketapi provides an extremely minimal Bitbucket API client. It
+// looks for a BITBUCKET_API_TOKEN environment variable to use authentication,
+// otherwise it relies on anonymous API access.
+package bitbucketapi
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"os"
+	"runtime"
+	"sync"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+type repository struct {
+	SCM string `json:"scm"`
+}
+
+const environmentVariable = "BITBUCKET_API_TOKEN"
```

Comment:
@kevinburke I was re-reading through this, do you think we should namespace this? `DEP_BITBUCKET_API_TOKEN`?

Reply:
I think it's OK... I would like to make it as easy to guess as possible and the odds of someone needing two different tokens in the same environment seems low.